client/config/dev.config.js

```diff
@@ -0,0 +1,5 @@
+const SECRETS = require('./secrets.json'); // nodejs will auto read json
```

Comment:
I'm assuming you made a client specific secrets.json file because this relative path doesn't look right? If that is the case, we should probably consider moving that file up to root, so its easily available to both front and backend paths.

Reply:
I agree it would be nice not to track two secrets.json files. I don't see a problem with moving it to the root directory unless that gets back to the comment about exposing sensitive data to the client. Another thing to factor is this component is going to use the api to access map data instead of accessing google directly so it won't need a secrets file at all when that happens.

Reply:
Good point - this looks good to me but (p)react is still new to me so I'll leave it to @motosharpley to do the review/merge. I played around with this a little further and we can actually use window.navigator.geolocation (https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation) to get user location as default instead of making api call to google maps geolocation. This will access natively the user location unless they don't allow us access. It has a watch method to get continuous update. I'll post the modified code in comments.

Reply:
That's a cool addition - thanks!